src/SilverStripe/BehatExtension/Context/BasicContext.php

```diff
@@ -800,4 +800,41 @@ public function theTextBeforeAfter($textBefore, $order, $textAfter, $element) {
 			assertTrue(strpos($text, $textBefore) > strpos($text, $textAfter));
 		}
 	}
+
+	/**
+	 * @When /^I scroll to page bottom$/
```

Comment:
Please change to a correct English sentence: "I scroll to the bottom of the page" and "I scroll to the top of the page"